google_apis.md

```diff
@@ -7,17 +7,30 @@ and use that project for all your environments.
 
 ![google_app_1](images/google_app_1.png)
 
-## Enable Google Maps JS APIs
+* After creating it, make sure that it is selected in the header and enable the Google APIs that you need for this project under `APIs & Services > Library`
 
-* When enabling the Google Maps Javascript API key you can generate one or more keys for your environments.
-What is important, is to restrict the usage of the key to certain domains.
+* When asked, attach the `wg-education CC` as billing account.
 
-![google_app_2](images/google_app_2.png)
+## API key generation
 
-Remember to enable the specific domain for production and both the renuoapp.ch and herokuapp.com domains.
+**Maps JavaScript API**
 
-* When asked, attach the `wg-education CC` as billing account.
+In order to user the Google APIs you will need to generate new credentials. Because we are using only one project per application, but would like to separate the usage in development from the one in production, name the keys like so: `MASTER Maps Public Key` and `DEVELOP Maps Public Key`.
+
+**Geocoding API**
+
+As for the JS API, create two different keys for the two environments, but this time replacing `Public` with `Private`. This is because the Geocoding API key won't be exposed in the frontend.
 
+## API key restrictions
+
+**Maps JavaScript API**
```

Comment:
let's add this part together with the maps section. So that we have a `Maps Javascript API` section which includes all the steps, and the same for the `Geocoding` part.